src/login/logind-dbus.c

```diff
@@ -1946,7 +1946,11 @@ static int send_prepare_for(Manager *m, const HandleActionData *a, bool _active)
 
         assert(m);
         assert(a);
-        assert(IN_SET(a->inhibit_what, INHIBIT_SHUTDOWN, INHIBIT_SLEEP));
+        assert(IN_SET(a->inhibit_what, INHIBIT_SHUTDOWN, INHIBIT_SLEEP, _INHIBIT_WHAT_INVALID));
```

Comment:
maybe move this line down, after the if check. that way you don't have to extend the list.